tests/parser.cpp

```diff
@@ -2731,7 +2731,7 @@ TEST(Parser, subprog_enum_arg)
 {
   test("fn f($a : enum x): void {}",
        "Program\n"
-       " f: void($a : enum x)\n");
+       " f: void($a : uint64)\n");
```

Comment:
Why the change here and below?

Reply:
Because of the parser change. This is no longer a record type but an enum.

Reply:
Hm, well it's not exactly an enum arg then. Maybe just delete these tests?

Reply:
this is probably more correct than it was - we probably just don't have a different to_string or whatever for ints that are enums.

Reply:
Nevermind - fixed this in the latest version.